ecs-cli/modules/cli/local/secrets/clients/clients.go

```diff
@@ -74,6 +82,17 @@ func (d *SSMDecrypter) DecryptSecret(arnOrName string) (string, error) {
 	return *val.Parameter.Value, nil
 }
 
+// Clean up param if it is hierarchical.
+// SSM parameters containing hierarchies must start with "/"
+// Leading / is optional for non-hierarchical param names
+func formatParam(p string) string {
```

Comment:
Would it make sense to just always include the forward slash? According to the [docs](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-parameter-name-constraints.html), based on the example given on Uniqueness it looks like having the leading slash would be treated the same as if it weren't there for a non-nested parameter.

Reply:
I think that makes sense, I'm just skittish of the possibility of this breaking for any edge case. Happy to be proven wrong, though.

Reply:
Fair -- is there manual output for re-running the command that surfaced this bug form #1010 ?